nixos/modules/services/networking/keybase.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.keybase;
+
+in {
+
+  ###### interface
+
+  options = {
+
+    services.keybase = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to start the Keybase service.";
+      };
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.keybase;
+        defaultText = "pkgs.keybase";
+        example = literalExample "pkgs.keybase";
+        description = "Keybase derivation to use.";
+      };
+
+    };
+  };
+
+  ###### implementation
+
+  config = mkIf cfg.enable {
+
+    systemd.user.services.keybase = {
+      description = "Keybase service";
+      serviceConfig = {
+        ExecStart = ''
+          ${cfg.package}/bin/keybase service --auto-forked
```

Comment:
I don't think it's necessary, really. I only chucked it in there as that's what you get when running from a shell - so, yeah we can probably drop this. I'll remove it and test it, along with all the other changes.